Update serval.py

hlruh opened the pull request:
Allows serval to read in sky mask via the command line option skymsk and other than the skymsk specified in the inst file.

hlruh committed:
Update serval.py

Allows serval to read in sky mask via the command line option skymsk and other than the skymsk specified in the inst file.

mzechmeister commented on src/serval.py:
```diff
@@ -1088,10 +1088,10 @@ def serval():
-
+         skyfile = getattr(inst, 'skyfile', None)
+   if skyfile:
+      sky = np.genfromtxt(servallib + skyfile)
```
It seems now  `skyfile` must be in servallib.

hlruh replied:
Skyfile needs to be in servallib, this ensures it is analogues to the keyword in inst, where the skyfile also needs to be in servallib. Alternatively, one could  first try to find the skyfile in servallib and otherwise interpret it as dir?

mzechmeister reviewed (changes requested):
How about just changing the indent of the last two lines:
```
    if skyfile:
      if skyfile=='auto' and getattr(inst, 'skyfile', None):
         skyfile = servallib + getattr(inst, 'skyfile', None)
      sky = np.genfromtxt(skyfile)
      skymsk = interp(lam2wave(sky[:,0]), sky[:,1])
```

hlruh committed:
Update serval.py

Follow change request by @mzechmeister and simply indent the lines where the skyfile is read in, such that external files can be read in.
